src/actions/channels.js

```diff
@@ -1283,6 +1283,18 @@ export function unfavoriteChannel(channelId) {
     };
 }
 
+export function updateChannelScheme(channelId, schemeId) {
+    return bindClientFunc(
+        async () => {
+            await Client4.updateChannelScheme(channelId, schemeId);
+            return {channelId, schemeId};
+        },
+        ChannelTypes.UPDATE_CHANNEL_SCHEME_REQUEST,
+        [ChannelTypes.UPDATE_CHANNEL_SCHEME_SUCCESS, ChannelTypes.UPDATED_CHANNEL_SCHEME],
+        ChannelTypes.UPDATE_CHANNEL_SCHEME_FAILURE,
+    );
+}
+
```

Comment:
@jespino Your question about `bindClientFunc` made me think of a cleaner way to write this. However, `bindClientFunc` strips out your data if you have the word `SUCCESS` in your action! So that was a little odd to debug.